src/redis/zmalloc.h

```diff
@@ -82,32 +84,32 @@
 #define HAVE_DEFRAG
 #endif
 
-void *zmalloc(size_t size);
-void *zcalloc(size_t size);
-void *zrealloc(void *ptr, size_t size);
-void *ztrymalloc(size_t size);
-void *ztrycalloc(size_t size);
-void *ztryrealloc(void *ptr, size_t size);
-void zfree(void *ptr);
+void* zmalloc(size_t size);
```

Comment:
I excluded the redis sources from the commit formatting checks so that we don't have to change it

Reply:
OK, I will do it it was auto format from the IDE